CloudFlare: add the configuration for logpush (#315)
SEKOIA-IO/documentation: docs/operation_center/integration_catalog/cloud_and_saas/cloudflare/cloudflare.md

- `Cloudflare Zone ID`
- `Cloudflare account email`

#### with logpush

!!! note

    It's the recommended way to get your events from Cloudflare.

> Configure a [logpull job](https://developers.cloudflare.com/logs/reference/logpush-api-configuration/) with the following destination:

Reviewer: ```suggestion
Configure a [Logpush job](https://developers.cloudflare.com/logs/reference/logpush-api-configuration/) with the following destination:
```

`https://intake.sekoia.io/plain/batch?header_X-SEKOIAIO-INTAKE-KEY=<your intake key>`

##### Manage logpush with cURL

```bash
$ curl -X POST https://api.cloudflare.com/client/v4/zones/<CLOUDFLARE_ZONE_ID>/logpush/jobs \
-H "X-Auth-Email: <CLOUDFLARE_EMAIL>" \
-H "X-Auth-Key: <CLOUDFLARE_API_KEY>" \
-H "Content-Type: application/json" \
--data '{
    "dataset": "http_requests",
    "enabled": true,
    "max_upload_bytes": 5000000,
    "max_upload_records": 1000,
    "destination_conf": "https://intake.sekoia.io/plain/batch?header_X-SEKOIAIO-INTAKE-KEY=<INTAKE_KEY>"
}'
```

#### with logpull

> You can now enable Cloudflare log retention (see [associated documentation](https://developers.cloudflare.com/logs/logpull/enabling-log-retention/)). 

Reviewer: I didn't get why this section start with this sentence, maybe you deleted something ?

Author: This phrase `You now have all information to configure the Cloudflare module and its `Send events` action to SEKOIA.IO.` was deleted

> You can now enable Cloudflare log retention (see [associated documentation](https://developers.cloudflare.com/logs/logpull/enabling-log-retention/)). 

Author: Here is a rewording suggestion
```suggestion
 To allow the pulling of logs, enable [Cloudflare log retention](https://developers.cloudflare.com/logs/logpull/enabling-log-retention/). 
```

Reviewer: Maybe

```suggestion
 To start using Logpull, start by enabling [Cloudflare log retention](https://developers.cloudflare.com/logs/logpull/enabling-log-retention/). 
```
